mmcv/cnn/bricks/conv.py

```diff
@@ -6,6 +6,7 @@
 CONV_LAYERS.register_module('Conv2d', module=nn.Conv2d)
 CONV_LAYERS.register_module('Conv3d', module=nn.Conv3d)
 CONV_LAYERS.register_module('Conv', module=nn.Conv2d)
+CONV_LAYERS.register_module('ConvTranspose2d', module=nn.ConvTranspose2d)
```

Comment:
I suggest to use `ConvTranspose2d` wrapper implemented in MMCV rather than `nn.ConvTranspose2d`. The wrapper performs normally as `nn.ConvTranspose2d` in normal cases and could handle empty tensors.

Reply:
fixed